django/db/migrations/autodetector.py

```diff
@@ -640,17 +690,25 @@ def generate_created_models(self):
                         )
                         for removed_base_field in removed_base_fields:
                             dependencies.append(
-                                (base_app_label, base_name, removed_base_field, False)
+                                OperationDependency(
+                                    base_app_label,
+                                    base_name,
+                                    removed_base_field,
+                                    DependencyType.REMOVE,
+                                )
                             )
             # Depend on the other end of the primary key if it's a relation
             if primary_key_rel:
                 dependencies.append(
-                    resolve_relation(
-                        primary_key_rel,
-                        app_label,
-                        model_name,
-                    )
-                    + (None, True)
+                    OperationDependency(
+                        *resolve_relation(
+                            primary_key_rel,
+                            app_label,
+                            model_name,
+                        ),
```

Comment:
This was only formatted like this by black because of the addition.

It now makes sense to flatten it since you're editing the lot:
 ```suggestion
                        *resolve_relation(primary_key_rel, app_label, model_name),
```